create.js

```diff
@@ -113,8 +117,7 @@ module.exports = function(_nativeProtocols) {
 
   // returns a safe copy of options (or a parsed url object if options was a string).
   // validates that the supplied callback is a function
-  function parseOptions (options, callback, wrappedProtocol) {
-    assert.equal(typeof callback, 'function', 'callback must be a function');
+  function parseOptions (options, wrappedProtocol) {
```

Comment:
No need for the callback to be passed to `parseOptions`, as it is not checked anymore.